tests/components/ozw/test_cover.py

```diff
@@ -0,0 +1,86 @@
+"""Test Z-Wave Covers."""
+from homeassistant.components.cover import ATTR_CURRENT_POSITION
+
+from .common import setup_ozw
+
+
+async def test_cover(hass, cover_data, sent_messages, cover_msg):
+    """Test setting up config entry."""
+    receive_message = await setup_ozw(hass, fixture=cover_data)
+    # Test loaded
+    state = hass.states.get("cover.roller_shutter_3_instance_1_level")
+    assert state is not None
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5760157)